{
  "bugzilla": {
    "description": "hdf5: multiple CVEs",
    "id": "2280037",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2280037"
  },
  "csaw": false,
  "details": [
    "HDF5 Library through 1.14.3 may use an uninitialized value in H5A__attr_release_table in H5Aint.c."
  ],
  "name": "CVE-2024-32611",
  "package_state": [
    {
      "cpe": "cpe:/a:redhat:openstack:16.1",
      "fix_state": "Out of support scope",
      "package_name": "hdf5",
      "product_name": "Red Hat OpenStack Platform 16.1"
    }
  ],
  "public_date": "2024-05-10T00:00:00Z",
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2024-32611\nhttps://nvd.nist.gov/vuln/detail/CVE-2024-32611\nhttps://www.hdfgroup.org/2024/05/new-hdf5-cve-issues-fixed-in-1-14-4/"
  ],
  "threat_severity": "Moderate",
  "upstream_fix": "hdf5 1.14.4"
}